python/py-tensorflow-metal/Portfile

```diff
@@ -39,11 +39,14 @@ platform i386 {
     set minimum_supported_major_version 20
 }
 
-if {${os.platform} eq {darwin} && ${os.major} < ${minimum_supported_major_version}} {
+if {!(${os.arch} in [list arm i386])
```

Comment:
this should/could become `(${os.arch} ni [list arm i386])`

Reply:
but regardless this will (I *think*) not solve the problem that was reported in the Trac ticket . It will continue to fail to parse on these platforms since the `if`-statement will be evaluated and `minimum_supported_major_version` is still not defined. So to me it seems your only solution here is to just define `minimum_supported_major_version` outside of the `platform` specifiers; for example you could set it to `20` and then only for `arm` set it to `21`. Then I don't think you'll need the `(${os.arch} ni [list arm i386])` check at all.

Reply:
> `ni`

Wow it was worth fixing this issue off only to learn about this obscure tcl operator. https://www.youtube.com/watch?v=zIV4poUZAQo

It will fix the issue because tcl does short-circuit evaluation.

Reply:
> It will fix the issue because tcl does short-circuit evaluation.

okay, that I didn't know ;) As a last, minor request can you please update the commit message - I'd think that saying "fix parsing error......" is more precise?

Reply:
Thanks! Done.